### DISCUSSION TITLE
Cannot share links for files

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [filebrowser/filebrowser#3269] [General]
Hello, I set 2FA trough Authentik server and now when I want to share some file user have to register Authentik account. Is there any workaround to share for anonymous persons?
Regards.

**Comment by andrew-manger** [comment]
I’m thinking you can make certain paths available without authentication. I use Authelia because it is simpler, but I required authentication for the base URL, but allow the /share/* and /dl/* paths to bypass authentication. However, when accessing the URLs, all I get is a file browser loading screen.

Edit: I actually figured it out (but I switched to the filebrowser quantum fork so I haven’t tested with is image). You have to allow /api/* and /static/* to bypass as well
